script/sheet/armor.js

```diff
@@ -4,6 +4,28 @@ export class ForbiddenLandsArmorSheet extends ForbiddenLandsItemSheet {
   static get defaultOptions() {
     return mergeObject(super.defaultOptions, {
       template: "systems/forbidden-lands/model/armor.html",
+      tabs: [
+        {
+          navSelector: ".sheet-tabs",
+          contentSelector: ".sheet-body",
+          initial: "attributes",
```

Comment:
```suggestion
          initial: "main",
```
It doesn't seem to make any difference, but to avoid confusion I think it's best to update this to the correct initial tab name. Also needed for weapon.js, I believe.

Reply:
Good catch! Will do:)